cmd/commands.go

```diff
@@ -15,285 +15,285 @@
 package caddycmd
 
 import (
-	"bytes"
-	"crypto/rand"
-	"encoding/json"
-	"flag"
-	"fmt"
-	"io"
-	"io/ioutil"
-	"log"
-	"net"
-	"net/http"
-	"os"
-	"os/exec"
-	"path/filepath"
-	"strings"
+    "bytes"
```

Comment:
I'm not sure why the spaces are differnt here.  Do I need to set some config in go fmt?

Reply:
Go fmt uses tabs. No configuration is necessary. Your editor can probably integrate with gofmt and run it on save.

Reply:
To clarify, the spaces will have to be converted back into tabs, before this is feasible to review.

Reply:
👍

Reply:
@mholt, this is fixed now.